test/runtests.jl

```diff
@@ -34,15 +34,15 @@ add_hyperedge!(h2;vertices=Dict(3:5 .=> (2.5,4.5,5.5)))
 
 
 
-b = BipartiteView{Float64}(h1)
+b = BipartiteView(h1)
 
 @test sum(LightGraphs.adjacency_matrix(LightGraphs.SimpleGraph(b))) == 18
 
 @test sort(collect(LightGraphs.outneighbors(b,5))) == [7,9]
 @test sort(collect(LightGraphs.outneighbors(b,1))) == [6]
 @test sort(collect(LightGraphs.inneighbors(b,9))) == [3,4,5]
 
-# == [4,3,5]
+@test Set(LightGraphs.vertices(b)) == Set(1:LightGraphs.nv(b))
```

Comment:
it would be probably simpler to do `using LightGraphs` above and not have to qualify function calls here.